docs/how-to/explore/find-any-resource.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_label: Find any resource
+sidebar_custom_props:
+  tags: [AWS, GCP, DigitalOcean, Kubernetes]
+---
+
+# How to quickly find any resource
+
+## Introduction
+
+Resoto maintains an asset inventory of your infrastructure metadata. It is equipped with a full-text search index that can be leveraged to find any resource, no matter which cloud provider, account, region or service the resource belongs to.
+
+Examples where this approach might be helpful:
```

Comment:
- no hedging ("might") - we need to be assertive. 
- be specific "this approach" --> which approach? 
- let's use full sentences

Here are three examples where full-text search is helpful: